composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "studiometa/ui",
-  "version": "0.2.18",
+  "version": "0.2.20",
```

Comment:
**issue:** The version should be `0.2.19` here 

I forgot a parameter with the NPM command I gave you @perruche, here is the one to use:

```sh
npm version -ws --include-workspace-root <version>
```

You can run it with `0.2.19` as parameter to make sure all version numbers are correct:

```sh
npm version -ws --include-workspace-root 0.2.19
```